earth_enterprise/docs/landing_page/manual/answer/6014171.html

```diff
@@ -17,79 +17,22 @@
 <a name="top_of_file"></a>
 <p><img src="../art/common/logo_col_126x48.png" width="126" height="48" alt="Google logo" /></p>
 <h1><a href="../answer/index.html">Google Earth Enterprise Documentation Home</a> | Developers</h1>
-<h2>Add Python Libraries to GEE Python</h2><p>Python v2.7.5 is bundled with GEE, providing functionality such as publishing and custom search plug-ins.</p>
+<h2>Add Python Libraries to GEE Python</h2>
+<p>GEE uses system Python with some extra libraries to support functionality such as publishing and custom search plug-ins.</p>
 
-<p>The standard Python v.2.7.5 libraries are included with GEE. Additional modules specific to GEE are installed in:</p>
+<p>Additional modules specific to GEE are installed in:</p>
 
-<p><code>/opt/google/gepython/Python-2.7.5/lib/python2.7/site-packages</code></p>
+<p><code>/opt/google/gepython/Python-2.7.x/lib/python2.7/site-packages</code> (where x is determined by the system Python version.</p>
 
 <ul>
   <li><code>psycopg2</code> is a PostgreSQL database adapter</li>
   <li><code>google/protobuf</code> is Google's data storage mechanism</li>
   <li><code>mgrs</code> provides conversion to and from MGRS</li>
-  <li><code>PIL</code> provides image processing and graphics capabilities, supporting many different file formats</li>
   <li><code>GDAL</code> (Geospatial Data Abstraction Library) Python library</li>
 </ul>
 
 <h3>Add a Python library</h3>
-<p>You add a Python library to <code>/opt/google/gepython/Python-2.7.5</code> for it to be enabled with GEE. One of the easiest ways to install a Python library is to use the Python package tool, <code>pip</code>. You may also need to consider any other dependencies for your newly installed library.</p>
+<p>You can add a Python library to either the system python path, or to the <code>/opt/google/gepython/Python-2.7.x</code> path for it to be enabled with GEE. One of the easiest ways to install a Python library is to use the Python package tool, <code>pip</code>. You may also need to consider any other dependencies for your newly installed library.</p>
```

Comment:
@asmitaw I was trying to make this about the path (e.g. `/usr/lib/python2.7/` - obviously system dependent though), not the product. Would `<code>python</code>` be acceptable to you?